components/organisms/Viewpdf/Viewpdf.module.css

```diff
@@ -0,0 +1,98 @@
+.viewpdf-container {
+    position: absolute;
+    height: 100vh;
+    width: 100vw;
+    background-image: url(https://i.imgur.com/UQqp7LT.png);
+    background-size: 100% 100%;
+    overflow-y: scroll;
+}
+
+.viewpdf-containerBackButton {
+    display: flex;
+    position: absolute;
+    font-size: var(--fontSize-titles_mobile);
+    z-index: 1;
+    top: 0.313rem;
+}
+
+.viewpdf-backButton {
+    display: flex;
+    align-items: center;
+    gap: 0.5rem;
+    text-decoration: none;
+    font-size: var(--fontSize-buttons_mobile);
+    transition: var(--transition);
+    background-color: transparent;
+    border: none;
+    cursor: pointer;
+}
+
+.viewpdf-backButton:hover {
+    color: var(--turquoise);
+}
+
+.viewpdf-title {
+    font-size: var(--fontSize-titles_mobile);
+    margin: 2.5rem auto 1.875rem auto;
+    position: relative;
+    display: flex;
+    justify-content: center;
+}
+
+.container-pdf {
+    position: relative;
+    margin: 0 auto;
+    width: 80%;
+    height: 29.375rem;
+}
+
+.pdf {
+    width: 100%;
+    height: 100%;
+}
+
+@media (width >=48rem) {
```

Comment:
From this measurement, the PDF should go to the end of the view

This is how it is:
![image](https://github.com/generaciondesafiante/Regresemos_a_casa_UI/assets/104586980/0349db08-ee8b-43af-b8e7-d7bef763b8d2)

This is how it should be:
![image](https://github.com/generaciondesafiante/Regresemos_a_casa_UI/assets/104586980/84f44b80-f3e1-4b16-ac25-9ce6af40316b)

Reply:
![image](https://github.com/generaciondesafiante/Regresemos_a_casa_UI/assets/104875565/5deed27b-560c-4876-80a5-14b8267342f3)
so ?